driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/fcntl.bpf.c

```diff
@@ -20,6 +20,7 @@ int BPF_PROG(fcntl_e,
 		return 0;
 	}
 
+	/// TODO: we can remove the size from here.
```

Comment:
So, can we drop all `SYSCALL_{E/X}_SIZE` defines?

Reply:
yeah this is exactly what I do in the next commit but only in `ringbuf__store_event_header` method, we need to keep the one in `ringbuf__reserve_space`